src/test/scala/uk/gov/hmrc/test/ui/pages/registration/RegistrationPage.scala

```diff
@@ -55,7 +55,7 @@ object RegistrationPage extends BasePage {
     }
 
     submitPage()
-    onPage(AmlAccountingActivityPage.heading)
+//    onPage(AmlAccountingActivityPage.heading)
```

Comment:
should this be commented out?

Reply:
I am reusing the code for the change AML supervisor link. So the landing page is different. Shall I make another method and test this instead of using the same?

Reply:
I have changed it now. So will remove the comment.